### PR TITLE
How we could load arbitrary tasks

### PR DESCRIPTION
**Pull request by kumare3**
# TL;DR
_Please replace this text with a description of what this PR accomplishes._

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

File: flytekit/core/python_function_task.py [hunk 1]
```diff
@@ -159,7 +159,79 @@ def istestfunction(func) -> bool:
...
+    @property
+    def task_loader(self):
+        return self._task_loader
+
+    def get_command(self, settings: SerializationSettings) -> List[str]:
+        """
+        NOTE: This command is different, it tries to retrieve the actual LHS of where this object was assigned, so that
+        the module loader can easily retreive this for execution - at runtime.
+        """
+        var = settings.get_instance_var(self)
```
Comment (EngHabu) [inline flytekit/core/python_function_task.py:212]: Should this be
```suggestion
        var = settings.get_instance_var(self.task_loader)
```

? or am I missing something?

File: flytekit/bin/entrypoint.py [hunk 2]
```diff
@@ -198,47 +199,49 @@ def _handle_annotated_task(task_def: PythonTask, inputs: str, output_prefix: str
...
-
-                    # TODO: Perhaps remove.  This is a workaround to an issue we perceived with limited entropy in
-                    # TODO: AWS batch array jobs.
-                    _flyte_random.seed_flyte_random(
-                        "{} {} {}".format(_random.random(), _datetime.datetime.utcnow(), job_index)
+            if not test:
+                if isinstance(task_def, PythonTask):
+                    _handle_annotated_task(task_def, inputs, output_prefix, raw_output_data_prefix)
+                elif isinstance(task_def, TaskLoader):
+                    _task_def = task_def.load_task(loader_args=loader_args)
```
Comment (EngHabu) [inline flytekit/bin/entrypoint.py:214]: So the idea is that task_module and task_name will refer to the TaskLoader instead? that's a bit odd I think... I think these should be new & different flags instead....

File: flytekit/core/python_function_task.py (hunk 1 above)
Comment (kumare3) [reply flytekit/core/python_function_task.py:212]: ya it should be you are right, if task_loader is set

**Comment by kumare3** [comment]
Closing the PR in favor fo @wild-endeavor's TaskResolver work

**Pull request closed without merging**